cedar-policy-core/src/entities/json/value.rs

```diff
@@ -94,6 +94,9 @@ pub enum CedarValueJson {
     Record(
         #[cfg_attr(feature = "wasm", tsify(type = "{ [key: string]: CedarValueJson }"))] JsonRecord,
     ),
+    /// JSON null, which is never valid, but we put this here in order to
+    /// provide a better error message.
+    Null,
```

Comment:
Does this just implicitly deserailize from null without additional annotation?

Reply:
It does seem so, as shown by this PR's tests